HW_5/Task2.Sum.py

```diff
@@ -0,0 +1,27 @@
+# Задача 28: Напишите рекурсивную функцию sum(a, b), возвращающую сумму двух целых неотрицательных чисел.
+# Из всех арифметических операций допускаются только +1 и -1. Также нельзя использовать циклы.
+# *Пример:*
+# 2 2 ->  4
+
+
+def sum(numA, numB):
+    if numA == 0:
+        return numB
+    if numA > 0:
+        return sum(numA-1, numB+1)
```

Comment:
2 условие не нужно, так как это все остальные случаи.
```suggestion
    if numA == 0:
        return numB

    return sum(numA - 1, numB + 1)
```